apps/token-e2e/src/integration/view/home.test.js

```diff
@@ -0,0 +1,96 @@
+import navigation from '../../locators/navigation.locators';
+import home from '../../locators/home.locators';
+
+context('Home Page - verify elements on page', function () {
+  before('visit token home page', function () {
+    cy.visit('/');
+  });
+
+  describe('with wallets disconnected', function () {
+    before('wait for page to load', function () {
+      cy.get(navigation.section, { timeout: 10000 }).should('be.visible');
+    });
+
+    describe('Navigation tabs', function () {
+      it('should have HOME tab', function () {
+        cy.get(navigation.section).within(() => {
+          cy.get(navigation.home).should('be.visible');
+        });
+      });
+      it('should have VESTING tab', function () {
+        cy.get(navigation.section).within(() => {
+          cy.get(navigation.vesting).should('be.visible');
+        });
+      });
+      it('should have STAKING tab', function () {
+        cy.get(navigation.section).within(() => {
+          cy.get(navigation.staking).should('be.visible');
+        });
+      });
+      it('should have REWARDS tab', function () {
+        cy.get(navigation.section).within(() => {
+          cy.get(navigation.rewards).should('be.visible');
+        });
+      });
+      it('should have WITHDRAW tab', function () {
+        cy.get(navigation.section).within(() => {
+          cy.get(navigation.withdraw).should('be.visible');
+        });
+      });
+      it('should have GOVERNANCE tab', function () {
+        cy.get(navigation.section).within(() => {
+          cy.get(navigation.governance).should('be.visible');
+        });
+      });
+    });
+
+    describe('THE $VEGA TOKEN table', function () {
+      it('should have TOKEN ADDRESS', function () {
+        cy.get(home.tokenDetailsTable).within(() => {
+          cy.get(home.address).should('be.visible');
+        });
+      });
+      it('should have VESTING CONTRACT', function () {
```

Comment:
suggestion (non-blocking): The team tends to use arrow functions
```suggestion
      it('should have VESTING CONTRACT', () => {
```

Reply:
I'm fine with that, but there is one thing we need to take into consideration when changing from 'regular' function to arrow functions. Explained here: https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Avoiding-the-use-of-this
FYI: @AndyWhiteVega @jtsang586

Reply:
Yeah i will have something to push up (hopefully today) that shows how THIS can simplify everything.
Especially in the end2end world, it is such a useful tool for throwing data about